code/modules/mob/living/status_procs.dm

```diff
@@ -241,6 +241,66 @@
 			P = apply_status_effect(STATUS_EFFECT_PARALYZED, amount)
 		return P
 
+///////////////////////////////// STARTLED //////////////////////////////////
+
+/mob/living/proc/IsStartled() //If we're Startled
+	return has_status_effect(STATUS_EFFECT_STARTLED)
+
+/mob/living/proc/AmountStartled() //How many deciseconds remain in our Startled status effect
+	var/datum/status_effect/incapacitating/startled/I = IsStartled()
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names